.changeset/tasty-colts-hug.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-azure': minor
+'@backstage/plugin-azure-backend': minor
+'@backstage/plugin-azure-common': minor
+---
+
+Generic azure plugin based on an annotation supplied by a given entity.
```

Comment:
Perhaps this could say a sentence or two about what this plugin does currently. Perhaps make sure that the description in `microsite/data/plugins/azure.yaml` also mentions that it only does functions.

Do you expect this plugin to display more information or be specific to azure functions?